noaa/track.py

```diff
@@ -0,0 +1,20 @@
+import logging
+
+
+async def sql(es, params):
+    logger = logging.getLogger(__name__)
+    logger.info("Provided params [%s].", params)
```

Comment:
I guess that this was needed during development. Can we remove it now?

Reply:
👍 I'm also removing it from the `eql` custom operation then.